.github/workflows/builder.yml

```diff
@@ -296,6 +296,10 @@ jobs:
             # Create version tag
             create_manifest "${docker_reg}" "${{ needs.init.outputs.version }}" "${{ needs.init.outputs.version }}"
 
+            # Create series version tag (e.g. 2021.6.x)
+            v="${{ needs.init.outputs.version }}"
+            create_manifest "${docker_reg}" "${v%.*}.x" "${{ needs.init.outputs.version }}"
```

Comment:
I suggest losing the `.x`, just `2021.6`.

Additionally, how would this work with beta & nightly builds?